support-frontend/assets/components/subscriptionCheckouts/cancellationPolicy.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import React from 'react';
+import Text from 'components/text/text';
+
+export const cancellationCopy = {
+  title: 'You can cancel any time',
+  body: 'There is no set time on your agreement with us so you can end your subscription whenever you wish',
+};
+
+const CancellationPolicy = ({ orderIsAGift }: {orderIsAGift?: boolean}) => (
+  <Text>
+    {!orderIsAGift && (
```

Comment:
Is the `<CancellationPolicy>` component ever passed the `orderIsAGift` prop? The one place I can see this component in use doesn't pass it down? https://github.com/guardian/support-frontend/pull/2465/files#diff-0fc926fce08f4328d136ea9cbd69ec76R15

Reply:
Good Call I've updated CancellationPolicy and orderIsAGift in another PR https://github.com/guardian/support-frontend/pull/2481